### ISSUE TITLE
[Egg Request]: Starrypy with Starbound

### ISSUE THREAD

**Issue by thakyZ** [parkervcp/eggs#1353]
### Service

Starbound

### Does this expand an already existing service

Yes

### Link to game

https://store.steampowered.com/app/211820/Starbound/

### Links for server downloads

https://github.com/StarryPy/StarryPy3k

### Links for the install docs

https://github.com/StarryPy/StarryPy3k/blob/master/README.md

**Comment by thakyZ** [comment]
I would like to mention StarryPy3k has to be run with Python3.6 and nothing higher as it will break with 3.7+

**Comment by parkervcp** [comment]
@thakyZ So if I read this right. It needs both starbound and python installed to run correct?

**Comment by thakyZ** [comment]
Yes and according to the authors it can run newer versions of Python3 but a lot of things break after 3.6
